### ISSUE TITLE
UI Login trouble / failing in 2.3.5

### ISSUE THREAD

**Issue by klaasjoerg** [Z-Wave-Me/zwave-smarthome#242]
Since 2.3.5 I found some temporary strange behaviour with browser logins denied.
Usually the first login e.g. after restarting zway works just fine. 
Some while later a new login results into authentication failure.

The z-wave-log shows the following output:

```
[2017-08-10 09:07:57.681] [E] [HTTP] Callback execution error: Error: Invalid digest
    at Error (native)
    at Object.crypto.sha512 (zway/crypto:25:51)
    at hashPassword (automation/Utils.js:72:51)
    at ZAutomationAPIWebRequest._.extend.verifyLogin (automation/ZAutomationAPIProvider.js:281:181)
    at ZAutomationAPIWebRequest.ZAutomationWebRequest.handleRequest (automation/WebserverRequestRouter.js:259:41)
    at automation/WebserverRequestRouter.js:32:35
    at WebServer.document_root (automation/Webserver.js:35:11)
```

Also homebridge is not able to login/authenticate anymore. The only way I found getting back in again is to restart the zway process.

**Comment by klaasjoerg** [comment]
@PoltoS :   I also noticed, that this is related with websocket connections being interrupted or disconnected. After that, no re-login seems to be possible.
see issue:  https://github.com/Z-Wave-Me/home-automation/issues/441

**Comment by mforss** [comment]
Further investigation shows that the login errors occurs after any of these actions:

1. Open websocket connection on non-existing host
2. Open websocket connection on host which is down
3. Close websocket connection after successful connection to any host

The only way to resolve the problem is to restart the zway server.

**Comment by PoltoS** [comment]
This problem occurs because OpenSSL have a pretty bad isolation of contexts in case of failures. We will focus this issue in next version

**Comment by PoltoS** [comment]
Closed with OpenSSL 1.1.0
